notify-send.sh

```diff
@@ -44,6 +57,7 @@ Application Options:
   -i, --icon=ICON[,ICON...]         Specifies an icon filename or stock icon to display.
   -c, --category=TYPE[,TYPE...]     Specifies the notification category.
   -h, --hint=TYPE:NAME:VALUE        Specifies basic extra data to pass. Valid types are int, double, string and byte.
+  -o, --action=NAME:ACTION           Specifies action button which should be integrated to the notification. NAME will be dispayed, ACTION is the comman to run.
```

Comment:
Specifying multiple actions is working by design, isn't it? I have tested and it worked for me.
Let's reflect this in command help and / or in example in README.md

Reply:
Well I reuse the same behavior that there is for the hints. I added